scripts/execute_tutor_extra_commands.sh

```diff
@@ -0,0 +1,2 @@
+# Populate the retirement states
+tutor local exec lms python manage.py lms populate_retirement_states
```

Comment:
Would it be possible to just run this within the script that executes the test command and skip this script entirely?
[https://github.com/eduNEXT/integration-test-in-tutor/pull/8/files#r183854869](https://github.com/eduNEXT/integration-test-in-tutor/pull/8/files#r1838548698)